go/consensus/tendermint/apps/scheduler/scheduler.go

```diff
@@ -194,9 +195,13 @@ func (app *schedulerApplication) BeginBlock(ctx *api.Context, request types.Requ
 		)
 
 		if entitiesEligibleForReward != nil {
-			accounts := publicKeyMapToSortedSlice(entitiesEligibleForReward)
+			accountIDs := []staking.ID{}
+			for _, entity := range publicKeyMapToSortedSlice(entitiesEligibleForReward) {
```

Comment:
Does it matter that we sort based on the public key, and not the account IDs?

Reply:
Agreed, I think it works both ways. I've changed the code in the file to use account addresses instead of entities' public keys in majority of the places.